lib/errors.ts

```diff
@@ -171,3 +170,29 @@ export async function handleError(error: any) {
 	// The exit error code was set above through `process.exitCode`.
 	process.exit();
 }
+
+export function printErrorMessage(message: string) {
+	const chalk = getChalk();
+	console.error(chalk.red(message));
+	console.error(chalk.red(`\n${messages.getHelp}\n`));
+}
+
+/**
+ * Print a friendly error message and exit the CLI with an error code, BYPASSING
+ * error reporting through Sentry.io's platform (raven.Raven.captureException).
+ * Note that lib/errors.ts provides top-level error handling code to catch any
+ * otherwise uncaught errors, AND to report them through Sentry.io. But many
+ * "expected" errors (say, a JSON parsing error in a file provided by the user)
+ * don't warrant reporting through Sentry.io.  For such mundane errors, catch
+ * them and call this function.
+ *
+ * DEPRECATED: Use `throw new ExpectedError(<message>)` instead.
+ */
+export function exitWithExpectedError(message: string | Error): never {
```

Comment:
As this function was moved to a different file and this PR was pending review while other PRs got merged, I suggest a new string search ("exitWithExpectedError") over the CLI codebase to make sure that no usage was left behind.